smaily_for_drupal/config/install/field.field.node.smaily_response_page.field_response_page_type.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  enforced:
+    module:
+      - smaily_for_drupal
+  config:
+    - field.storage.node.field_response_page_type
+    - node.type.smaily_response_page
+id: node.smaily_response_page.field_response_page_type
+field_name: field_response_page_type
+entity_type: node
+bundle: smaily_response_page
+label: 'Success page'
```

Comment:
This label is not descriptive enough. Use something like "Choose response page type".